app/views/shared/authentication/_email_login_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_for(User.new, as: :user, url: session_path(:user)) do |f| %>
+<%= form_for(User.new, as: :user, url: session_path(:user), data: { testid: "login-form" }) do |f| %>
```

Comment:
Going with `data-testid`. This is common practice and we do this already in our other tests. For more information, see [Making your UI tests resilient to change](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change).